xbmc/rendering/RenderSystem.cpp

```diff
@@ -47,26 +45,9 @@ void CRenderSystemBase::GetRenderVersion(unsigned int& major, unsigned int& mino
   minor = m_RenderVersionMinor;
 }
 
-bool CRenderSystemBase::SupportsNPOT(bool dxt) const
+bool CRenderSystemBase::SupportsNPOT() const
 {
-  if (dxt)
-    return (m_renderCaps & RENDER_CAPS_DXT_NPOT) == RENDER_CAPS_DXT_NPOT;
-  return (m_renderCaps & RENDER_CAPS_NPOT) == RENDER_CAPS_NPOT;
-}
-
-bool CRenderSystemBase::SupportsDXT() const
-{
-  return (m_renderCaps & RENDER_CAPS_DXT) == RENDER_CAPS_DXT;
-}
-
-bool CRenderSystemBase::SupportsBGRA() const
-{
-  return (m_renderCaps & RENDER_CAPS_BGRA) == RENDER_CAPS_BGRA;
-}
-
-bool CRenderSystemBase::SupportsBGRAApple() const
-{
-  return (m_renderCaps & RENDER_CAPS_BGRA_APPLE) == RENDER_CAPS_BGRA_APPLE;
+  return true;
```

Comment:
it's not always true for windows (see RenderingSystemDX for details). so at least we must make this method virtual.